miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridgeDispatcher.kt

```diff
@@ -34,6 +35,7 @@ interface UserInfoBridgeDispatcher {
     /** Get access token from host app. **/
     fun getAccessToken(
         miniAppId: String,
+        accessTokenPermission: AccessTokenPermission,
```

Comment:
We will be releasing this in 3.1.0, so we need to avoid making this a breaking change. I'm not sure of the best way to handle this. 

I think that `AccessTokenPermission` is an important access control measure, so basically we don't want the Access Tokens to be retrievable at all if the Host App hasn't implemented `AccessTokenPermission`. So maybe we can just keep both function signatures and add `@Deprecated` to the old one?

```
@Deprecated
fun getAccessToken(
    miniAppId: String,
    onSuccess: (tokenData: TokenData) -> Unit,
    onError: (message: String) -> Unit
) {
    throw MiniAppSdkException("The `UserInfoBridgeDispatcher.getAccessToken` $NO_IMPL")
}

fun getAccessToken(
    miniAppId: String,
    accessTokenPermission: AccessTokenPermission,
    onSuccess: (tokenData: TokenData) -> Unit,
    onError: (message: String) -> Unit
) {
    throw MiniAppSdkException("The `UserInfoBridgeDispatcher.getAccessToken` $NO_IMPL")
}
```

What do you think?

Reply:
Also, is it possible to share this feature in the release note?

Reply:
@corycaywood @khairul-alam-licon I have updated with deprecated one + release note + userguide.